frontend/src/app/containers/Restart.js

```diff
@@ -37,7 +37,7 @@ export default class Restart extends React.Component {
       <div className="full-page-wrapper space-between">
         <Header forceHideSettings={ false } />
         <div className="centered-banner restart-message">
-          <h2 data-l10n-id="restartRequiredHeader">Test Pilot Installed!</h2>
+          <p data-l10n-id="restartRequiredSubHeader">Almost done . . s.</p>
```

Comment:
This doesn't match the string in the localization file. Also, let's use `&hellip;` here, too.